.github/workflows/build.yml

```diff
@@ -51,14 +58,22 @@ jobs:
         arguments:
 
   rust-check-git-rev-deps:
+    if: github.event_name != 'schedule'
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v3
     - uses: stellar/actions/rust-check-git-rev-deps@main
 
   build:
     runs-on: ubuntu-latest
-
+    env:
+      CACHED_PATHS: |
```

Comment:
the ci script [purges the cache from time to time](https://github.com/stellar/stellar-core/blob/be1b3ee138ff75c42bfae8c90cec111feffad785/ci-build.sh#L131) (on top of ccache being configured to not grow indefinitely).
How can we make sure that this cargo cache does not grow too much?
On my dev box (before running some `cargo cache` commands, my cache was in the GBs)

Reply:
Ok I've added code to both incrementally clear rust caches and join in the purge-once-a-month ccache party. This requires installing two (!) separate cargo extensions, one to manage its downloads (cargo-cache) and another to manage its build artifacts (cargo-sweep).